client/src/app/ui/modules/qr-code/components/qr-code/qr-code.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { QrCodeComponent } from './qr-code.component';
+
+xdescribe(`QrCodeComponent`, () => {
+    let component: QrCodeComponent;
+    let fixture: ComponentFixture<QrCodeComponent>;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            declarations: [QrCodeComponent]
+        }).compileComponents();
+
+        fixture = TestBed.createComponent(QrCodeComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it(`should create`, () => {
+        expect(component).toBeTruthy();
+    });
+});
```

Comment:
Please write a test that checks if the qr code contains data.

Reply:
Can't do that because the test environment is in conflict with the HTMLCanvasElement that is used by that component